### PR TITLE
Manta Fix - Bridge Intercoms & Enginge Speakers

### LINKED ISSUE #927: Manta loudspeakers and bridge intercoms are misaligned
**Describe the bug**
Bridge intercoms should be a bit to the north so they aren't on the floor. Loudspeakers should actually be touching walls.
image

### PR DESCRIPTION
**Pull request by MarkNais**
[MINOR]
## About the PR 
Moved the intercoms on Manta's bridge to be a bit more visually pleasing.
Added auto pixel shifting to the speakers (engine rooms),
Minor refactor of pixel-placement of intercoms - I was initially pursuing a different method of fixing speakers, but didn't like it. So the `pixel_place()` is a left-over; I can remove it and return it to being in `New()`.
image
image
image
Notably, this patch goes with the offset of the red intercom (the one above the character). Others are shown incase "we" want something else.

## Why's this needed? 
Fixes #927

### CODE REVIEW

File: code/obj/item/device/radio.dm [hunk 1]
```diff
@@ -1081,6 +1081,22 @@ obj/item/device/radio/signaler/attackby(obj/item/W as obj, mob/user as mob)
...
+		if(src.pixel_x == 0 && src.pixel_y == 0)
+			switch(src.dir)
+				if(NORTH)
+					pixel_y = -14
+				if(SOUTH)
+					pixel_y = 32
+				if(EAST)
+					pixel_x = -21
+				if(WEST)
+					pixel_x = 21
```
Comment (ZeWaka) [inline code/obj/item/device/radio.dm:1098]: yeah rename this or just move it to new imo

Comment (MarkNais) [reply code/obj/item/device/radio.dm]: Done.

File: code/obj/item/device/radio.dm [hunk 2]
```diff
@@ -1081,6 +1081,19 @@ obj/item/device/radio/signaler/attackby(obj/item/W as obj, mob/user as mob)
 	density = 0
 	desc = "A Loudspeaker."
 
+	New()
+		//..()
```
Comment (ZeWaka) [inline code/obj/item/device/radio.dm:1085]: why no `..()`?

Comment (MarkNais) [reply code/obj/item/device/radio.dm:1085]: The parent `loudspeaker` currently has a deliberately empty `New()`.
In the case that the loudspeaker has a "populated" new in the future, it seems weird to me that the speakers should inherit it i.m.o.

Comment (ZeWaka) [reply code/obj/item/device/radio.dm:1085]: weird, and that doesn't call parents either.

**Pull request merged**